torchvision/prototype/transforms/color.py

```diff
@@ -0,0 +1,116 @@
+import collections.abc
+import functools
+from typing import Any, Dict, Union, Tuple, Optional, Sequence, Callable
+
+import PIL.Image
+import torch
+from torch import nn
+from torchvision.prototype import features
+from torchvision.prototype.transforms import Compose
+from torchvision.prototype.transforms import Transform, functional as F
+
+from ._utils import is_simple_tensor
+
+
+class ColorJitter(Transform):
+    def __init__(
+        self,
+        brightness: Optional[Union[float, Sequence[float]]] = None,
+        contrast: Optional[Union[float, Sequence[float]]] = None,
+        saturation: Optional[Union[float, Sequence[float]]] = None,
+        hue: Optional[Union[float, Sequence[float]]] = None,
```

Comment:
@FeryET I'm aware you wanted to work on this, but I needed `ColorJitter` for something else. Thus, I went ahead and implemented it. Could you check if the signature is now sufficient for what you were asking in #5646? I also adapted the `_check_input` method to reflect the changes.